src/runtime/debug/debug.go

```diff
@@ -58,3 +61,10 @@ type Module struct {
 func SetGCPercent(n int) int {
 	return n
 }
+
+// String implements Stringer for BuildInfo.
+//
+// Not implemented.
+func (bi *BuildInfo) String() string {
+	return "<build info placeholder>\n"
```

Comment:
Can you implement this?

Reply:
given everything else is empty in the struct, what would that do?

could we just import the real code from big go?

Reply:
"else"?  It's not printing anything at all.  It could at least return "go. " + GoVersion, couldn't it?

Reply:
ic the first line sure, I didn't notice because when I use buildinfo I print the GoVersion separately let me add something

btw should it be
```
go	tinygo0.33.1
```
or
```
tinygo tinygo0.33.1
```

Reply:
doing the former, copying the code

Reply:
@dkegel-fastly so I copied the real code but... it's ugly... what is the general philosophy/must not be the first time tinygo is just a small variant of an existing go file, can you import+patch or some other way to avoid copy pasta (this is more for my education than anything specific for this PR - hopefully it's ok now with https://github.com/tinygo-org/tinygo/pull/4343/commits/41dd04a20b5675741156621549c3c9df2900e3f9 )